cookbooks/ambari-server/metadata.rb

```diff
@@ -0,0 +1,9 @@
+name             'ambari-server'
+maintainer       'YOUR_COMPANY_NAME'
```

Comment:
Can we put some information here? You can refer [this](https://github.com/amithkanand/chef-bach/blob/master/cookbooks/kafka-bcpc/metadata.rb) to get some idea.